server/player.lua

```diff
@@ -36,12 +36,38 @@ function QBCore.Player.Login(source, citizenid, newData)
     end
 end
 
+function QBCore.Player.GetOfflineUser(citizenid)
```

Comment:
I think it's more fitting if you named it `GetOfflinePlayer`

Reply:
Renamed! Good callout!